python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -957,6 +987,32 @@ def _get_repo_job_by_name(context: SensorEvaluationContext, job_name: str) -> Jo
 
         return resolved_run_requests
 
+    def resolve_backfill_requests(
```

Comment:
nit, but this isn't really modifying the run requests at all, so it might be more accurate to call this "validate_backfill_requests"